content/billing/billing.md

```diff
@@ -29,7 +29,7 @@ Click on **Update billing details** to change the billing information or add add
 
 ### Changing card
 
-To change the card used for payments, click **Edit** in the credit card section under the billing details. You can then click **Add new card** to register a new card or select a different card from already registered cards.
+To change the card used for payments, click **Edit** in the credit card section under the billing details. You can then click **New card** to register a new card or select a different card from already registered cards.
```

Comment:
Anything in **New card** refers to the actual text in our application, we can't change this here unless we change the app. So worth reverting anything inside ** throughout this change set.